onnx/test/shape_inference_test.py

```diff
@@ -383,6 +384,18 @@ def test_expand_dynamic_shape(self) -> None:
             graph,
             [make_tensor_value_info('y', TensorProto.INT32, (None, 2, None))])
 
+    def test_expand_symbolic_shape(self) -> None:
+        graph = self._make_graph(
+            [('x', TensorProto.INT32, (1, 2, None)),
+             ('shape', TensorProto.INT64, ('unk__0',))],
```

Comment:
I believe that this also needs to be `('shape', TensorProto.INT64, None)`, right?